tests/webcompat-tests.js

```diff
@@ -0,0 +1,20 @@
+const expect = require('chai').expect
+const fs = require('fs')
+
+describe('WebCompat Settings Tests', () => {
+    const Ajv = require('ajv').default
+    const ajv = new Ajv()
+    const rootSchema = JSON.parse(fs.readFileSync('./tests/schemas/webcompat-settings.json', 'utf8'))
```

Comment:
Should this not be checking the file output for each one? We're doing overrides already which isn't captured here.

Reply:
I can change it to just check the output of macos build instead